Dockerfile

```diff
@@ -3,6 +3,17 @@ FROM python:3.7-slim
 # Set working directory to root
 WORKDIR /root
 
+# Install additional helper packages
+RUN apt-get update && apt-get install -y \
```

Comment:
What instances are using the packages for? For our use case I don't think we edit the images much.